repos/system_upgrade/el7toel8/actors/migratesendmail/actor.py

```diff
@@ -21,14 +23,20 @@ def process(self):
         if not decision or not decision.migrate_files:
             return
 
+        migrated = []
         for f in decision.migrate_files:
-            migratesendmail.migrate_file(f)
+            if os.path.exists(f):
+                migratesendmail.migrate_file(f)
+                migrated.append(f)
```

Comment:
@PeterMocary  thinking about that, add also error log when a file cannot be migrated, instead of skipping it only.

Reply:
Added.